commandapi-platforms/commandapi-bukkit/commandapi-bukkit-test/commandapi-bukkit-test-tests/src/test/java/dev/jorel/commandapi/CommandAPIVersionHandler.java

```diff
@@ -43,12 +43,12 @@ private static boolean isMojangMapped() {
 		}
 	}
 	
-	static CommandAPIPlatform<?, ?, ?> getPlatform() {
+	static LoadContext getPlatform() {
 		if(profileId == null) {
 			System.out.println("Using default version 1.19.4");
-			return new MockNMS(new NMS_1_19_4_R3());
+			return new LoadContext(new MockNMS(new NMS_1_19_4_R3()), () -> {});
```

Comment:
This could use the default `LoadContext` constructor
```suggestion
			return new LoadContext(new MockNMS(new NMS_1_19_4_R3()));
```